cmd/starlark/starlark.go

```diff
@@ -24,6 +24,7 @@ import (
 var (
 	cpuprofile = flag.String("cpuprofile", "", "gather CPU profile in this file")
 	showenv    = flag.Bool("showenv", false, "on success, print final global environment")
+	execprog   = flag.String("c", "", "execute prog `c`")
```

Comment:
Er, you changes program to prog but left c alone. :)

Reply:
Sigh. I'm so scatterbrained right now. Multitasking. Sorry. :( Fixed now.

Reply:
No worries. The "Commit" button on most of these apps might as well be called "Begin spelling review" in my case.